app/page/network/connections.js

```diff
@@ -0,0 +1,49 @@
+const { h, computed, throttle } = require('mutant')
+
+module.exports = function Connections ({ localPeers, connectedPeers, avatar }) {
+  const state = buildState({ localPeers, connectedPeers })
+
+  return [
+    {
+      title: 'local peers',
+      body: h('LocalPeers', [
+        computed(state.localPeers, peers => {
+          if (!peers.length) return h('p', 'No local peers (on same wifi/ LAN)')
+
+          return peers.map(peer => avatar(peer))
+        }),
+        h('p', [
+          h('i.fa.fa-info-circle'),
+          'these are people on the same WiFi/ LAN as you right now. You might not know some of them yet, but you can click through to find out more about them and follow them if you like.'
+        ])
+      ])
+    },
+    {
+      title: 'remote peers',
+      body: h('RemotePeers', [
+        computed(state.remotePeers, peers => {
+          if (!peers.length) return h('p', 'No remote peers connected')
+
+          return peers.map(peer => avatar(peer))
+        }),
+        h('p', [
+          h('i.fa.fa-info-circle'),
+          'these are people on the same WiFi/ LAN as you right now. You might not know some of them yet, but you can click through to find out more about them and follow them if you like.'
```

Comment:
This should be a different text (remote peers), right?

Reply:
nice catch!